gipc/gipc.py

```diff
@@ -56,6 +57,18 @@
 log = logging.getLogger("gipc")
 
 
+# Python 3.8 changed the default subprocess method from 
+# fork to spawn. Spawn does not copy any state from the
+# old process. So we should force forking.
```

Comment:
> So we should force forking

The key argument seems to be missing here. For example, on Windows, for gipc is has _always_ been the case that the child process was not a fork, but a fresh process created with win32 `CreateProcess()` -- basically the same as now for macOS with `spawn()`. What really _is_ the key argument for why we want to switch back to `fork()` on macOS?

Reply:
On 3.7 and below, the default for python on osx and linux was consistently fork... Since we do a lot of dev work on osx, but execute on linux in production, the change now causes inconsistent behavior for us in development.

We could argue this should be the concern of the application, and not gipc though. And make gipc work with either popen method.

I'd be ok with that and just moving this to our application.

Reply:
> I'd be ok with that and just moving this to our application.

Thanks for doing that; the right choice I think!